DuplicateAssemblyScanner/DuplicateAssemblyScanner/Settings.cs

```diff
@@ -0,0 +1,124 @@
+namespace DuplicateAssemblyScanner {
+    using ColossalFramework.UI;
+    using DuplicateAssemblyScanner.Structs;
+    using DuplicateAssemblyScanner.Util;
+    using ICities;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Generates the settings screen based on cached results from the assembly scanner.
+    /// </summary>
+    public class Settings {
+        // Markers used in checkbox labels
+        private const string MARKER_BLANK = " ";
+        private const string MARKER_ENABLED = "*";
+
+        // Cache of scan results
+        private static Dictionary<string, int> duplicates_;
+        private static Dictionary<string, List<ModAssembly>> occurrences_;
+        private static bool duplicatesFound_;
+
+        /// <summary>
+        /// Generate the options screen listing all the duplicates (if found).
+        /// </summary>
+        /// 
+        /// <param name="helper">UI helper for building settings screen.</param>
+        public static void CreateUI(UIHelperBase helper) {
```

Comment:
The .NET style guide suggests using the BSD (Allman) style of the braces: the opening brace must be placed on an empty line.

Reply:
Will deal with this as part of #19 at later date.